tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_axis_utils.ts

```diff
@@ -168,3 +168,48 @@ export function getTicksForLinearScale(
 
   return {major: Array.from(majorTickValMap.values()), minor};
 }
+
+const canvasForMeasure = document.createElement('canvas').getContext('2d');
+
+export function filterTicksByVisibility(
```

Comment:
Could you add docstring to explain this function along with the parameters? 
I do not fully understand them until read the tests. (especially `getDomPos` and `marginBetweenAxis`)